.github/ISSUE_TEMPLATE/new_cheatsheet_proposal.md

```diff
@@ -1,17 +1,32 @@
 ---
-name: new_cheatsheet_proposal
-about: Used to create a proposal to add a new cheat sheet to the project
-title: New cheat sheet proposal
+name: New Cheat Sheet Proposal
+about: Used to create a proposal to add a new cheat sheet to the project.
+title: New cheat sheet proposal: [PUT_TARGET_CS_NAME_HERE]
```

Comment:
Let's change cheat sheet to CS. Makes it smaller and is actually understandable.

Reply:
Done in a754728.